package.json

```diff
@@ -3,9 +3,12 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.html",
+  "homepage": "https://eliotc1986.github.io/gulp-webpack-demo/",
   "scripts": {
     "start": "gulp",
-    "build": "gulp build --production"
+    "build": "gulp build --production",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d dist"
```

Comment:
This tells github to prepare the dist folder and production assets before deploy. The second step tells it to deploy the dist folder